blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.155",
+  "version": "0.0.155-Thor2",
```

Comment:
?

Reply:
Ok, @scherler explained me offline why this is so weird.

Reply:
@michaelneale are we on time to backport this to 1.2.x?

Reply:
@alvarolobato probably not (yes versions are weird) - some conflicts to resolve

Reply:
Again :( @scherler  can you have a look?